benchmarks/dynamo/check_hf_bert_perf_csv.py

```diff
@@ -16,7 +16,8 @@ def check_hf_bert_perf_csv(filename):
     for _, row in df.iterrows():
         model_name = row["name"]
         speedup = row["speedup"]
-        if speedup < 1.19:
+        threshold = 1.19 if model_name != "hf_Bert" else 1.18
```

Comment:
```suggestion
        # this can be bumped up in future when performance gets back
        threshold = 1.15
```

Reply:
Seems 1.17 is sufficient so far: https://github.com/pytorch/pytorch/actions/runs/4160700392/jobs/7198177291#step:11:1588